stackslib/src/chainstate/stacks/boot/pox_4_tests.rs

```diff
@@ -1234,3 +1235,36 @@ fn balances_from_keys(
         .map(|principal| get_stx_account_at(peer, tip, &principal))
         .collect()
 }
+
+#[test]
+fn get_current_signer_set() {
+    let (epochs, pox_constants) = make_test_epochs_pox();
+    let mut burnchain = Burnchain::default_unittest(
+        0,
+        &BurnchainHeaderHash::from_hex(BITCOIN_REGTEST_FIRST_BLOCK_HASH).unwrap(),
+    );
+    burnchain.pox_constants = pox_constants.clone();
+    let (mut peer, keys) =
+        instantiate_pox_peer_with_epoch(&burnchain, function_name!(), Some(epochs.clone()), None);
+
+    match peer.sortdb.take() {
+        Some(sortdb) => {
+          println!("SortitionDB readwrite: {}", sortdb.readwrite);
+          println!("SortitionDB first_block_height: {}", sortdb.first_block_height);
+          println!("SortitionDB first_burn_header_hash: {:?}", sortdb.first_burn_header_hash);
+          println!("SortitionDB pox_constants: {:?}", sortdb.pox_constants);
+
+        let tip = get_tip(Some(&sortdb));
+
+        let tip_index_block = tip.get_canonical_stacks_block_id();
+        let reward_cycle = burnchain
+            .block_height_to_reward_cycle(tip.block_height)
+            .unwrap() as u64;
+        peer.chainstate()
```

Comment:
This doesn't test anything.  There's nothing in this data map, so there's nothing to check. Please actually add data to the data map and verify that you can read it out.